src/View/Miller.vala

```diff
@@ -47,10 +47,15 @@ namespace Files.View {
             }
         }
 
-        public Miller (GLib.File loc, View.ViewContainer ctab, ViewMode mode) {
-            this.ctab = ctab;
-            this.root_location = loc;
+        public Miller (GLib.File loc, View.ViewContainer _ctab, ViewMode _mode) {
```

Comment:
It looks like this takes an argument `_mode` that is never used?

Reply:
True - its always been like that I think.  I guess the idea was to allow the possibility of a multi-slot view with the individual slots showing grid or list views instead or column views.  In the event, that was never implemented and the slots are hard-coded as column views so I guess that parameter is  redundant.  I'll remove it.